### ISSUE TITLE
have you computed the EER?

### ISSUE THREAD

**Issue by numanzafar** [gzerveas/mvts_transformer#35]
Thank you for your complete code with the detailed description.
After completing the Training, have you computed/ calculated the Equal Error Rate using the classification dataset by using the pre-trained model?

**Comment by gzerveas** [comment]
No, I haven't, because the goal of the evaluation was to compare with other state-of-the-art methods, and this metric is not typically reported.  But I also don't understand how it is applicable here - the Equal Error Rate can be used only in cases of binary classification. At any rate, you have access to the prediction scores, so you can certainly use e.g. sklearn to get the AUROC, EER, etc.
